file_store/src/file_store.rs

```diff
@@ -180,11 +180,13 @@ impl FileStore {
         let client = self.client.clone();
         infos
             .map_ok(move |info| get_byte_stream(client.clone(), bucket.clone(), info.key))
+            .fuse()
             .try_buffered(2)
             .flat_map(|stream| match stream {
                 Ok(stream) => stream_source(stream),
                 Err(err) => stream::once(async move { Err(err) }).boxed(),
             })
+            .fuse()
```

Comment:
should this stream be fused in two different places?

Reply:
Removed spurious fuse here: d0f21234652f4287f74164343b5b2afd1e774384

get_byte_stream now has its own fuse